src/mesh/voxel_cloud.rs

```diff
@@ -448,6 +448,26 @@ impl ScalarField {
         tools::weld(&joined_voxel_mesh, (min_voxel_dimension as f32) / 4.0)
     }
 
+    /// Compute bounding box from scalar field. The bounding box will contain
+    /// the entire block of the voxel space described by the current scalar
+    /// field, regardless if it contains any voxels.
+    ///
+    /// The Bounding box will be defined in cartesian units.
+    pub fn bounding_box_cartesian(&self) -> BoundingBox<f32> {
```

Comment:
The other function returning a bounding box is called. `volume_voxel_space_bounding_box`. Either rename this to `cartesian_bounding_box` or the other to `bounding_box_volume_voxel_space`.

I usually slightly prefer the C-style naming of things (`object_verb` or `module_object_verb`), but decide what you think is more pleasing to the eye for you :slightly_smiling_face:

Reply:
Also. Do you think we should be using `cartesian_space` instead of `cartesian` to make the duality with `voxel_space` more obvious? I am undecided.